samples/dynamicfs/dynamic_fs.go

```diff
@@ -256,6 +256,10 @@ func (fs *dynamicFS) ReadFile(
 func (fs *dynamicFS) ReleaseFileHandle(
 	ctx context.Context,
 	op *fuseops.ReleaseFileHandleOp) error {
+	if op.OpContext.Pid == 0 {
+		return fuse.EINVAL
+	}
```

Comment:
Why are we doing this inconsistently only in a few sample fs'?

From a sanity point of view
- All sample fs will have this check for all operations
- One sample fs will have this check for all its operation (memfs)

Reply:
Eitherway, lets submit a PR to upstream and handle thier feedback because some of ^ is purely a matter of taste

Reply:
Thanks for pointing this out. I was trying to cover all operations/all implementations of the FileSystem interface and didn't pay enough attention on what samples I am changing. Cleaned it up to have everything in memfs. We have few uncovered operations, but I guess that's okay.